composer/utils/ddp.py

```diff
@@ -1,15 +1,52 @@
 # Copyright 2021 MosaicML. All Rights Reserved.
 
+import os
 import warnings
+from typing import Callable, Optional
 
 import torch.distributed as dist
 
 
-def get_global_rank() -> int:
+def _get_distributed_config_var(env_var: str,
+                                human_name: str,
+                                default: int,
+                                fetch_fn: Optional[Callable[[], int]] = None) -> int:
```

Comment:
See comment below re: torch.distributed not being available.

```suggestion
                                fetch_fn: Optional[str] = None) -> int:
```